src/js/components/atoms/tooltip/tooltip.ts

```diff
@@ -0,0 +1,20 @@
+import { Vue, Component, Prop } from 'vue-property-decorator';
+
+@Component({
+    name: 'Tooltip',
+    template: require('./tooltip.html'),
+})
+
+export default class Tooltip extends Vue {
+    @Prop({ type: String, required: false }) block?: string;
+
+    classObject() {
```

Comment:
If you make this a getter you get responsiveness as a bonus (and  you can omit the parenthesis after classObject)